src/graphql/mutations/__tests__/CreateReply.js

```diff
@@ -83,6 +83,9 @@ describe('CreateReply', () => {
 
     // Cleanup
     await client.delete({ index: 'replies', type: 'doc', id: replyId });
+
+    // refresh must be invoked before deleteByQuery, or the query may find nothing and delete nothing
+    await client.indices.refresh({ index: 'urls' });
```

Comment:
Previously, `client.deleteByQuery` may delete nothing when `urls` index is not refreshed. This causes unit tests to fail if `CreateReply` unit test is executed before other unit tests that reads `urls` index.

This manual refresh should make sure unit tests gets consistent results.